podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -8,31 +7,53 @@
 #
 FROM registry.access.redhat.com/ubi9/ubi:9.2
 
-ARG GO_VERSION="1.20.10"
-ARG PROTOC_VERSION="3.11.4"
+ARG ARCH="s390x"
+ARG PACKER_VERSION="v1.9.4"
+ARG PLATFORM="Linux"
```

Comment:
This is really needs to be a Variable ? maybe we can use it as fixed value?

Reply:
agree